cmd/plume/release.go

```diff
@@ -518,25 +518,62 @@ func modifyReleaseMetadataIndex(spec *fcosChannelSpec, commitId string) {
 		plog.Fatalf("unmarshaling release metadata json: %v", err)
 	}
 
-	url, err := url.Parse(fmt.Sprintf("https://%s.s3.amazonaws.com/prod/streams/%s/builds/%s/release.json", spec.Bucket, specChannel, specVersion))
+	releasePath := filepath.Join("prod", "streams", specChannel, "builds", specVersion, "release.json")
```

Comment:
Note we'll need to tweak this soon once we [fix the layout](https://github.com/coreos/coreos-assembler/issues/463#issuecomment-506011148).

Reply:
I'm not sure I understand, are we planning on moving the `release.json` from `s3://fcos-builds/prod/streams/$stream/builds/$version/`?

Reply:
Sorry, I was just confused. Don't mind me!